data/remnant missions.txt

```diff
@@ -495,3 +560,759 @@ mission "Remnant: Technology Available"
+			`	"Everything in here we have either learned how to duplicate in its current form, or have salvaged in sufficient quantities that we might as well be manufacturing it."`
+			`	She nods at a rack of Korath Heat Shunts while she explains, then turns back to you.`
+			`	"Of course the really advanced and rare stuff is still going to the labs first, and won't be available until we have learned all we can from it." She looks around proudly. "We don't have the industrial capacity or the numbers of researchers to keep up with the Alphas, the Korath, or whoever else might be out there. But we are very good at figuring out what others have built and learning from it.`
+			`	Any salvage or unrecognized technology gets scanned in our quarantine area for threats, then looked over to figure out if there is anything new in it. If there is something new, it is shipped to a lab for our researchers to dissect. Anything we have already figured out ends up in here, where Remnant captains are welcome to use it as they see fit."`
+			choice
+				`	"Threats?"`
+					goto threats
+				`	"Interesting."`
+			`	"Yes, it is, isn't it? So many new things out there to discover and learn about.`
+				goto end
+			label threats
+			`	"Yes, sometimes salvage that is brought in is dangerous. System Cores come with fleets of micro-bots, for instance. If they aren't de-activated properly, they keep trying to rebuild their surroundings in the form of the last ship they were configured to serve. And while it hasn't happened yet, we are worried that someday we might run into someone who builds traps into their equipment, or uses nanobots that could run amok. We haven't seen any of that yet, but our science-fiction stories are filled with tales of nanotech apocalypses and things like that.`
+			label end
+			`	Well, I need to get back to work. That last battle put us behind schedule on our repairs. Have a good day!" With a final flourish that you think is the equivalent of an exclamation mark, Tali strides out of the outfitters, heading towards another landing bay.`
+
+event "Remnant Salvage Available"
+	outfitter "remnant"
+		"Generator (Furnace Class)"
+		"Generator (Candle Class)"
+		"Triple Plasma Core"
+		"Small Heat Shunt"
+		"Large Heat Shunt"
+		"Systems Core (Small)"
+		"Thruster (Asteroid Class)"
+		"Thruster (Planetary Class)"
+		"Steering (Asteroid Class)"
+		"Steering (Planetary Class)"
+		"Thruster (Comet Class)"
+		"Steering (Comet Class)"
+		"Korath Fire-Lance"
+	
+
+mission "Remnant: Expanded Horizons Quarg 1"
+	name "Visiting the Quarg"
+	description "Take Remnant researchers to see some Quarg."
+	minor
+	passengers 3
+	blocked "This mission requires bunks for three passengers."
+	source
+		government "Remnant"
+	destination "Wayfarer"
```

Comment:
Wayfarer doesn't have Quarg. Echo does, or perhaps Lagrange, or Garkhord, or Forpelog, but not Wayfarer. Wayfarer is simply a human world in proximity to the Quarg. Notice how it doesn't trigger the first contact mission for the Quarg, but all the other worlds I mentioned do. (Humanika also doesn't trigger the first contact mission, as it is a "Quarg" world, but it's entirely inhabited by humans.)

It looks like this mission is going to need to be rewritten to compensate that change.

Reply:
I did kind of account for that with the last bit of the conversation: 

> "She eagerly accepts, although she expresses a preference for landing on the human world, so they can observe the Quarg ships from a distance, as they would rather not draw any attention to themselves."

She doesn't actually want to meet the Quarg, she just wants to see their ships from a distance. Given that they share a system and *can* breath that atmosphere (at least for short durations), I think it is reasonable that there may be some Quarg on Wanderer. Not living there, more like traders or diplomats or whatnot just there for a few hours on business. Given that we can see Quarg ships landing on the planet while in space, it makes sense that they might be equally visible on the ground.

Reply:
As with the other comment, I just re-read the planet description for Wayferer, and it says specifically that there are Quarg walking around and mingling with the humans there. 

> Some of the landing pads are designed with docking tubes specifically for Quarg visitors, and the ceilings are unusually high to accommodate the Quarg, several of whom are walking around and mingling with the humans here.

Reply:
I'll take it, but I would find it more appropriate to go directly to the source (i.e. an actual Quarg planet/station). :)